test/games_statistics_test.rb

```diff
@@ -0,0 +1,64 @@
+require "./test/test_helper"
+
+
+class GamesStatisticsTest < MiniTest::Test
+
+  def setup
+    game_path = './data/games.csv'
+    team_path = './data/teams.csv'
+    game_teams_path = './data/game_teams.csv'
+
+    locations = {
+      games: game_path,
+      teams: team_path,
+      game_teams: game_teams_path
+    }
+
+    @games_statistics = GamesStatistics.from_csv(locations)
```

Comment:
This should shift to what I wrote above:

```ruby
require "./test/test_helper"

class TeamStatisticsTest < Minitest::Test
  def setup
    game_path = './data/games.csv'
    team_path = './data/teams.csv'
    game_teams_path = './data/game_teams.csv'

    locations = {
      games: game_path,
      teams: team_path,
      game_teams: game_teams_path
    }
    @stat_tracker = StatTracker.from_csv(locations)
    @games = @stat_tracker.games
    @teams = @stat_tracker.teams
    @game_teams = @stat_tracker.game_teams

    @team_statistics = TeamStatistics.new(@games, @teams, @game_teams)
  end
```
To have it mimic the relationship between these classes.